vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler.go

```diff
@@ -136,6 +136,28 @@ func ValidateVPA(vpa *vpa_types.VerticalPodAutoscaler, isCreate bool) error {
 			if policy.ContainerName == "" {
 				return fmt.Errorf("containerPolicies.ContainerName is required")
 			}
+
+			// check that perVPA is on if being used
+			if err := validatePerVPAFeatureFlag(&policy); err != nil {
+				return err
+			}
```

Comment:
This piece of code made me wonder if we should try move closer to how k/k validates resources. That's something for another day though